openassessment/xblock/config_mixin.py

```diff
@@ -21,16 +22,6 @@
 }
 
 
-def import_waffle_switch():
```

Comment:
Can we add a custom attribute in a separate PR/version to ensure this is not used by edx-platform before removing.  In other words, deprecate first, check, then remove.